### PR TITLE
@W-9953126@ Release Notes for 2.12.0

### PR DESCRIPTION
**Pull request by ranekere-sfdc**

### CODE REVIEW

File: docs/_articles/en/release-information.md [hunk 1]
```diff
@@ -3,7 +3,29 @@ title: Release Information
...
+* Updated PMD to 6.38.0 and RetireJS to 2.2.5
+
+**Merged pull requests:**
+
+- @W-9930749@: Reverting the heartbeat scheduling and timeout to the de… [\#546](https://github.com/forcedotcom/sfdx-scanner/pull/546)
+- @W-9930749@: Fixing error in dedup key. [\#545](https://github.com/forcedotcom/sfdx-scanner/pull/545)
+- @W-9930749@: Added dedup key to Github action. [\#544](https://github.com/forcedotcom/sfdx-scanner/pull/544)
+- @W-9930749@: Cancelled, i.e. timed-out, heartbeat should now cause pa… [\#543](https://github.com/forcedotcom/sfdx-scanner/pull/543)
+- @W-9930749@: Added a 60-minute timeout to the heartbeat action. [\#542](https://github.com/forcedotcom/sfdx-scanner/pull/542)
+- @W-9930749@: Added a 60-minute timeout to the heartbeat action. [\#540](https://github.com/forcedotcom/sfdx-scanner/pull/540)
```
Comment (jfeingold35) [inline docs/_articles/en/release-information.md:23]: These are all relating to the heartbeat, and probably don't need to be in our release notes.

File: docs/_articles/en/release-information.md [hunk 2]
```diff
@@ -3,7 +3,29 @@ title: Release Information
...
+- @W-9846051@: Github Action heartbeat script now runs against release … [\#536](https://github.com/forcedotcom/sfdx-scanner/pull/536)
```
Comment (jfeingold35) [inline docs/_articles/en/release-information.md:24]: Also heartbeat, so probably doesn't need to be in our release notes.

File: docs/_articles/en/release-information.md [hunk 3]
```diff
@@ -3,7 +3,29 @@ title: Release Information
...
+- @W-9879038@: Updated PMD to 6.38.0. [\#534](https://github.com/forcedotcom/sfdx-scanner/pull/534)
+- @W-9791463@: Updated RetireJS to 2.2.5, and updated locally-stored vu… [\#533](https://github.com/forcedotcom/sfdx-scanner/pull/533)
+- @W-9879056@: Publishing can now only happen on branches named v2-X-Y,… [\#532](https://github.com/forcedotcom/sfdx-scanner/pull/532)
```
Comment (jfeingold35) [inline docs/_articles/en/release-information.md:27]: This only matters to people who can publish, which is us. So we probably don't need this in the release notes.

File: docs/_articles/en/release-information.md (hunk 1 above)
Comment (rmohan20) [reply docs/_articles/en/release-information.md:23]: We decided a couple of releases ago that we'll account for all the PRs here. The `Release Summary` above would list the changes that a user may care about.

**Review by rmohan20 — APPROVED** [review]

File: docs/_articles/en/release-information.md (hunk 1 above)
Comment (ranekere-sfdc) [reply docs/_articles/en/release-information.md:23]: Yes, I agree @rmohan20  that way we can be consistent.

**Pull request merged**